numba/tests/test_operators.py

```diff
@@ -1304,11 +1311,13 @@ def test_mod(self):
         self.run_arith_binop(self.op.mod_usecase, 'mod', samples)
 
     def test_pow(self):
+        if utils.PYVERSION == (3, 11):
+            extra_cast = dict(force_type=float)
```

Comment:
Is an `else` branch needed to define `extra_cast` else this will fail on < 3.11 with a `NameError`?

Another option to "fix" this would be to just restrict the input on 3.11 to numerically smaller values. As noted in the comments, there are already challenges present in the inputs/testing/implementation.

Reply:
oops, i'll fix that. and check if smaller numbers make a easier fix.